android/aepsdk-bom/sdk-bom/build.gradle

```diff
@@ -110,6 +110,10 @@ task bom_release_notes() {
         def markdownTable = StringBuilder.newInstance()
         def oldBomVersion = rootProject.bomVersion.toString()
         def newBomVersion = nextBomVersion()
+        markdownTable <<= "This BOM ([Bill of Materials](https://central.sonatype.com/artifact/com.adobe.marketing.mobile/sdk-bom)) release includes changes to the following Android extensions. \n"
```

Comment:
Can you remove the white space at the end of the lines? It will trigger a lint error in our docs repo.

Reply:
They are removed.